### PR TITLE
Add multitenancy integration test scenarios for extension module

### PR DESCRIPTION
**Pull request by ahmetoz**
## Todos: 
- [x] Ensure metadata object in the test cases.

### CODE REVIEW

File: extension/test/integration/make-payment.handler.spec.js [hunk 1]
```diff
@@ -1,97 +1,112 @@
 const { expect } = require('chai')
 
 const ctpClientBuilder = require('../../src/ctp')
-const iTSetUp = require('./integration-test-set-up')
-const constants = require('../../src/config/constants')
 const config = require('../../src/config/config')
+const constants = require('../../src/config/constants')
+const iTSetUp = require('./integration-test-set-up')
+
+describe('::make-payment with multiple store use case::', () => {
```
Comment (lojzatran) [inline extension/test/integration/make-payment.handler.spec.js:8]: Are you talking about testing with this endpoint? https://docs.commercetools.com/api/projects/stores

File: extension/test/integration/make-payment.handler.spec.js [hunk 2]
```diff
@@ -1,97 +1,112 @@
...
-        returnUrl: 'https://your-company.com/',
-      }
-      const paymentDraft = {
-        amountPlanned: {
-          currencyCode: 'EUR',
-          centAmount: 1000,
-        },
-        paymentMethodInfo: {
-          paymentInterface: constants.CTP_ADYEN_INTEGRATION,
+      await makePayment({
```
Comment (lojzatran) [inline extension/test/integration/make-payment.handler.spec.js:35]: I think this way it's better: https://stackoverflow.com/questions/17569382/parametrized-tests-with-mocha. We already used it in https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/test/e2e/credit-card.spec.js#L42

**Review by lojzatran — CHANGES_REQUESTED** [review]

**Comment by ahmetoz** [comment]
@lojzatran 
Is this something new or flaky? It showed up when I merged changes on the `multitenancy-notification` branch.

Screenshot 2021-02-03 at 13 47 57

File: extension/test/integration/make-payment.handler.spec.js (hunk 2 above)
Comment (ahmetoz) [reply extension/test/integration/make-payment.handler.spec.js]: Imo here it's better to run them in parallel in one test server and one compact case, I make calls parallel: 
https://github.com/commercetools/commercetools-adyen-integration/pull/539/commits/7a7c6039e219ec19f8892f0a001becb0b88aa713

**Comment by lojzatran** [comment]
> @lojzatran
> Is this something new or flaky? It showed up when I merged changes on the `multitenancy-notification` branch.
> 
> Screenshot 2021-02-03 at 13 47 57

I think it's flaky. I fixed this error already.

File: extension/test/integration/make-payment.handler.spec.js (hunk 1 above)
Comment (ahmetoz) [reply extension/test/integration/make-payment.handler.spec.js]: https://github.com/commercetools/commercetools-adyen-integration/pull/539/commits/7f2a208927784544332c79a28a52376eedc09f00

**Review by lojzatran — APPROVED** [review]

**Pull request merged**